.github/workflows/build-linux.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       run: |
            cd $GITHUB_WORKSPACE/build
            export PATH=$PATH:/qt/5.13.2/gcc_64/bin/
+           export QML_SOURCES_PATHS=$GITHUB_WORKSPACE/build/Main
```

Comment:
This shouldn't be set to anything in the build directory; it should just be wherever the `qml.qrc` file is, which is `(project root directory)/Main` in this case.

Reply:
I'll make the change on web and see if it works...